### ISSUE TITLE
accessibility clean up

### ISSUE THREAD

**Issue by lpgoldstein** [DOI-ONRR/nrrd#3714]

**Comment by lpgoldstein** [comment]
Went through completed onrr.gov accessibility issues. Still no fully compliant pages but page violations for top onrr.gov pages went from 42 to 7 violations overall. issues exist for each violation.
